### ISSUE TITLE
Basic data structure for libraries

### ISSUE THREAD

**Issue by Anglepi** [Anglepi/My-Many-Reads#34]
Create a data structure for the libraries, that will consist of:

- A name
- An owner
- A set of entries.

Each of these entries will consist of:

- A book
- A score
- A reading status

**Comment by JJ** [comment]
An issue should
1. Refer to an existing user story
2. Present an issue, a problem to solve. It does not add value to the client to decide, in advance, what kind of data structure should be used.

**Comment by JJ** [comment]
Please reformulate and reopen.

**Comment by JJ** [comment]
>     * A reading status

If you don't provide a user story for an issue, and this user story has some relationship with the business logic, it's impossible to test if this specific feature is needed, and if it is, what kind of values should it take. You've solved it with 5 values, of which `ON HOLD` I have no idea what it means, for instance. No idea either on what transitions are possible (Can you pass from "DROPPED" to "ON HOLD"?)

In summary, this is a justification of why there seems to be some justification for either opening again Milestone 1 (as requested in #92 ), or create a transitional milestone with user stories and issues not addressed in that and previous milestones before continuing to others.

Additionally, as far as I can tell, that code was not part of a pull request, so I don't think it was properly cleared.
